### PR TITLE
Implemented awesome guidelines

### PR DESCRIPTION
**Pull request by praveen-klaytn**
# Resolves

1. Resolved awesome-lint issues
2.  Followed awesome-list guidelines (badge, list formatting, creative commons license) - https://github.com/sindresorhus/awesome/blob/main/create-list.md
3. Added code of conduct, license and contribution files

### CODE REVIEW

**Review by kjhman21 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -37,228 +37,227 @@ This page introduces all links related to core and ecosystem development of Klay
...
+  - [DSRV](https://www.allthatnode.com/klaytn.dsrv)
+  - [Endpoint status page from Community](https://3794.github.io/klaytn-rpc-list/)
  
 ## Oracles
-* [Witnet](https://feeds.witnet.io/klaytn)
-* [SupraOracles](https://data.supraoracles.com/networks/klaytn)
-* [KlayOrale](https://www.klayoracle.com/)
+- [Witnet](https://feeds.witnet.io/klaytn)
+- [SupraOracles](https://data.supraoracles.com/networks/klaytn)
+- [KlayOrale](https://www.klayoracle.com/)
```
Comment (toniya-klaytn) [inline README.md:193]:
```suggestion
- [KlayOracle](https://www.klayoracle.com/)
```

File: README.md [hunk 2]
```diff
@@ -37,228 +37,227 @@ This page introduces all links related to core and ecosystem development of Klay
...
-* https://klaytn01.fautor.app
-* https://klaytn02.fautor.app
-* https://klaytn03.fautor.app
-* https://klaytn04.fautor.app
-* https://klaytn05.fautor.app
-* https://cypress.fautor.app/archive
-* https://klaytn-mainnet-rpc.allthatnode.com:8551
-* https://klaytn-rpc.gateway.pokt.network
+- For more details, please refer to [Klaytn Docs](https://docs.klaytn.foundation/dapp/json-rpc/public-en)
+- https://public-en-cypress.klaytn.net
```
[the remark this answers is not in the document]
Comment (praveen-klaytn) [reply README.md:57]: @toniya-klaytn ,  have removed the fautor links now

File: README.md [hunk 3]
```diff
@@ -37,228 +37,227 @@ This page introduces all links related to core and ecosystem development of Klay
...
+- https://public-node-api.klaytnapi.com/v1/cypress
+- https://klaytn01.fautor.app
```
[the remark this answers is not in the document]
Comment (praveen-klaytn) [reply README.md]: removed fantrie service nodes

**Review by toniya-klaytn — APPROVED** [review]

**Pull request merged**